template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hdfs/HdfsRoleConfigProvider.java

```diff
@@ -59,12 +59,22 @@ protected List<ApiClusterTemplateConfig> getRoleConfigs(String roleType, Templat
                     );
                 }
                 return List.of();
+            case HdfsRoles.GATEWAY:
```

Comment:
@ankitsinghal Is the role correct? Should it be DATANODE?

Reply:
Yes @kkalvagadda1, this is correct, we need to set this on the HDFS GATEWAY role only in client configs.